routes/payments.js

```diff
@@ -2,25 +2,36 @@ require('dotenv').config();
 const { STRIPE_PRIVATE_API_KEY = "missing_key" } = process.env;
 const stripe = require('stripe')(STRIPE_PRIVATE_API_KEY);
 const paymentsRouter = require("express").Router();
+const { isLoggedIn } = require("./util")
 
-const { YOUR_DOMAIN = 'http://localhost:3000' } = process.env;
-
-// TBD: modify to check isLoggedIn
-// Try it out: curl -X POST http://localhost:4000/api/payments/create-checkout-session
-paymentsRouter.post('/create-checkout-session', async (req, res) => {
+paymentsRouter.post('/create-checkout-session', isLoggedIn, async (req, res) => {
```

Comment:
do we want to allow guests to checkout? (wondering about `isLoggedIn`)

Reply:
also, we should have `const { DOMAIN_URL = 'http://porto3000.herokuapp.com/' } = process.env;`
And everyone should add this to their `.env`:
`DOMAIN_URL=http://localhost:3000`
Then update lines 17 and 18 to use `DOMAIN_URL` instead of `YOUR_DOMAIN`.